interfaces/cython/cantera/reactor.pyx

```diff
@@ -873,6 +903,27 @@ cdef class ReactorNet:
         """
         return self.net.step()
 
+    def get_estimate(self, double t, int k=-1):
```

Comment:
could be removed from cython (although needed internally)

Reply:
I think this should definitely be removed from Cython. Can you also add documentation to the C++ function that indicates the function should not be used to get an estimate of the solution a long time after the time the derivatives were calculated, and if users want the solution at a future time they should just integrate? And also that the function is deliberately not exposed in Cython? I want to make sure that our future selves know why that function is not in Cython, and that it was an intentional choice not a mistake 😄

Reply:
`get_estimate` always pulls the most recent derivative from CVODE. I moved the function to the `protected` section and left comments that they are deliberately not exposed.